src/code/z_actor.c

```diff
@@ -2932,11 +2938,13 @@ void func_800328D4(GlobalContext* globalCtx, ActorContext* actorCtx, Player* pla
     sp84 = player->unk_664;
 
     while (actor != NULL) {
-        if ((actor->update != NULL) && ((Player*)actor != player) && CHECK_FLAG_ALL(actor->flags, ACTOR_FLAG_0)) {
+        if ((actor->update != NULL) && ((Player*)actor != player) &&
+            CHECK_FLAG_ALL(actor->flags, ACTOR_FLAG_TARGETABLE)) {
```

Comment:
targetable encompasses the whole mechanism of navi flying over something and the player being able to target it. 
the CANT_TARGET flag makes it so you cant target it, but navi can still fly over it. I agree that its confusing but didnt have any other way to phrase it
a better way to phrase it might be CANT_LOCK_ON or something, but people would generally call that targeting

Reply:
maybe "target_focus_only"? something that makes it sound like it's taking a specific thing away compared to the targeting flag alone

Reply:
i would argue focus sounds like the targeting/locking on part, which is what is being disabled here